include/gz/math/Stopwatch.hh

```diff
@@ -28,7 +28,9 @@ namespace gz
   namespace math
   {
     // Use a steady clock
-    using clock = std::chrono::steady_clock;
+    // This alias is now deprecated; please use std::chrono::steady_clock
+    // directly instead.
+    using clock GZ_DEPRECATED(8) = std::chrono::steady_clock;
```

Comment:
windows CI doesn't like this

~~~
Stopwatch.hh(33,17): error C2062: type 'int' unexpected
~~~

Reply:
trying `[[deprecated]]` based on https://stackoverflow.com/a/50882131